tests/index.ts

```diff
@@ -153,32 +173,44 @@ export const linterFmtTest = ({
   preCheck?: TestCallback;
   postCheck?: TestCallback;
 }) => {
-  // Step 1: Detect test files to run
+  // Step 1a: Detect test files to run and versions for asserts.
   const linterTestTargets = detectTestTargets(dirname, namedTestPrefixes);
 
   describe(`Testing formatter ${linterName}`, () => {
-    // Step 2: Define test setup and teardown
-    const driver = setupDriver(dirname, {}, linterName, preCheck);
-
-    // Step 3: Asynchronously run each test
     linterTestTargets.forEach(({ prefix, inputPath }) => {
-      it(prefix, async () => {
-        const testRunResult = await driver.runFmtUnit(inputPath, linterName);
-        expect(testRunResult).toMatchObject({
-          success: true,
+      // Step 1b: Detect versions to test against if PLUGINS_TEST_LINTER_VERSION=Snapshots
+      const linterVersions = getVersionsForTest(dirname, linterName, prefix, "fmt");
+      linterVersions.forEach((linterVersion) => {
+        // TODO(Tyler): Find a reliable way to replace the name "test" that doesn't violate snapshot export names.
```

Comment:
Ah, I take it you did want to put the version in the describe name and then ran into this?

Reply:
Yep. It gets written to the snapshot, which is deterministic when you're running with `PLUGINS_TEST_LINTER_VERSION=Snapshots`, but not otherwise, since we don't know the version that's going to be enabled yet, causing it to be undefined--and quite annoying.